eslint.config.mjs

```diff
@@ -169,6 +169,12 @@ export default tseslint.config(
           ignorePrimitives: true,
         },
       ],
+      '@typescript-eslint/no-require-imports': [
+        'error',
+        {
+          allow: ['/package\\.json$'],
```

Comment:
@JoshuaKGoldberg I just wanted to ask a question.

The recent lint errors point to - 
1. https://github.com/typescript-eslint/typescript-eslint/blob/59bbf417702b53fe78d8a190da7054bc6c22b780/packages/website/sidebars/sidebar.rules.js#L1
2. https://github.com/typescript-eslint/typescript-eslint/blob/59bbf417702b53fe78d8a190da7054bc6c22b780/packages/website/src/clientModules.js#L4
  
& a few other places.

Should I add the regexes in this file 
or
should I add an eslint comment directive ?

Reply:
I'd say the inline config comments strategy. Config files in `packages/website` are pretty different from the rest of the codebase and tend to do some wacky things.